gum/backend-qnx/gummemory-qnx.c

```diff
@@ -0,0 +1,215 @@
+/*
+ * Licence: wxWindows Library Licence, Version 3.1
+ */
+
+#include "gummemory.h"
+
+#include "gumqnx-priv.h"
+
+#include <fcntl.h>
+#include <gio/gio.h>
+#include <stdio.h>
+#include <sys/mman.h>
+#include <sys/procfs.h>
+
+static gboolean
```

Comment:
I will land this now and fix this for you so we can get this in before the weekend.